docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -115,6 +115,14 @@ There you can find links to upgrade notes for other versions too.
     +           - method: setSqlLoggerFacade
     +           - method: setEntityManager
     ```
+- set `symfony/monolog-bundle` as conflicting in your `composer.json` and update it in your vendor ([#1148](https://github.com/shopsys/shopsys/pull/1148))
```

Comment:
```suggestion
- set `symfony/monolog-bundle` as conflicting in your `composer.json` and run `composer update` command ([#1148](https://github.com/shopsys/shopsys/pull/1148))
```

Reply:
What about our current user, that are not upgrading to latest version? Should we somehow let them now about this problem?

Reply:
dont know. i think that @Seldaek should release the new patch version or give us some advice first how to solve that parsing problem